src/plugins/graph/components/graph.tsx

```diff
@@ -272,6 +287,33 @@ export const Graph = observer(
               onRequestRowHeight={onRequestRowHeight}
             />
           }
+          {
+            showEditableGraphValue &&
+            axes.map((axis: AxisPlace, idx) => {
+              const axisModel = graphModel?.getAxis(axis);
+              const minVal = isNumericAxisModel(axisModel) ? axisModel.min : kDefaultNumericAxisBounds[0];
+              const maxVal = isNumericAxisModel(axisModel) ? axisModel.max : kDefaultNumericAxisBounds[1];
+              return (
+                <div key={`${axis}-min-max`}>
+                  <EditableGraphValue
+                    value={minVal}
+                    minOrMax={"min"}
+                    axis={axis}
+                    onValueChange={(newValue) => handleMinMaxChange("min", axis, newValue)}
+                    readOnly={readOnly}
+                  />
+                  <EditableGraphValue
+                    value={maxVal}
+                    minOrMax={"max"}
+                    axis={axis}
+                    onValueChange={(newValue) => handleMinMaxChange("max", axis, newValue)}
+                    readOnly={readOnly}
+                  />
+                </div>
+              );
+
+            })
+          }
```

Comment:
In my last PR review I asked why this was being rendered separately from `renderGraphAxes()`. Can you either move this into that function or explain why that wouldn't work?

Reply:
Yeah I originally tried that where `<EditableGraphValue> `is being returned in `renderGraphAxes()` But because it sits inside of this `<svg>` block (See below) 
<img width="773" alt="image" src="https://github.com/concord-consortium/collaborative-learning/assets/521934/81f77286-d1ea-4bdf-a60c-ed828079af29">

`<GraphAxis>`  renders correctly since it returns` <g>` and` <rect>` elements. 
However `<EditableGraphValue>` is wrapped by a `<div>` it runs into issues. Last week I also tried taking <GraphAxis> out  which didn't work and it just seemed like too much to even refactor so I didn't go down that path.